config/remotePlugin.js

```diff
@@ -128,6 +132,16 @@ module.exports = {
         },
       },
     },
+    {
+      type: 'core.page/route',
+      properties: {
+        path: '/app-studio/application-environment-details',
```

Comment:
HAC Core now support routes with params. So maybe we can use that instead of query params here - 
```suggestion
        path: '/app-studio/applications/:appName/environments/:envName',
```
PS: We also need to update the current application route to support route param. Something like - `/app-studio/applications/:appName`